Firestore/core/src/firebase/firestore/local/proto_sizer.mm

```diff
@@ -50,13 +51,17 @@ size_t DocumentKeyByteSize(const DocumentKey& key) {
     : serializer_(serializer) {
 }
 
-size_t ProtoSizer::CalculateByteSize(const MaybeDocument& maybe_doc) const {
-  size_t count = DocumentKeyByteSize(maybe_doc.key());
+int64_t ProtoSizer::CalculateByteSize(const MaybeDocument& maybe_doc) const {
+  int64_t count = DocumentKeyByteSize(maybe_doc.key());
   count += [[serializer_ encodedMaybeDocument:maybe_doc] serializedSize];
```

Comment:
Unrelated question: why is the size of the key added to the size of the document? Does the encoded representation of the document not include the key?